testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/management/deployments/EjbInvocationStatisticsTestCase.java

```diff
@@ -107,11 +108,20 @@ private static Boolean getEnableStatistics(final ManagementClient managementClie
         final ModelNode address = PathAddress.pathAddress(PathElement.pathElement(SUBSYSTEM, SUBSYSTEM_NAME)).toModelNode();
         address.protect();
         final ModelNode value = readAttribute(managementClient, address, "enable-statistics");
-        if (value.isDefined())
-            return value.asBoolean();
+
+        if (value.isDefined()) {
+            ModelNode result = resolveExpression(managementClient, value);
+            return result.asBoolean();
+        }
         return null;
     }
 
+    private static ModelNode resolveExpression(ManagementClient managementClient, ModelNode value) throws IOException {
```

Comment:
you should not have to call a separate operation to resolve the expression: the `readAttribute` method just need to pass the `resolve-expressions` to `true`.

Reply:
please see https://github.com/wildfly/wildfly/pull/12032#issuecomment-460224198

Reply:
I tried to run CLI command and I got the same error as mentioned in the above comment.

If this should be working we have bug to fix :)

Reply:
The key change is this:
```diff
-        final ModelNode value = readAttribute(managementClient, address, "enable-statistics");
+        final ModelNode value = readAttribute(managementClient, address, "statistics-enabled");
```
With this change `resolve-expressions` works like a charm.
Test case was using obsolete attribute name.